Algorithm Codes/Graph/AdjacencyListGraph.py

```diff
@@ -150,6 +150,34 @@ def floyd_warshall(self):
                 return None
         
         return dist
+    
+    def prims(self):
+        nodes_left = self.n
+        visited = [False for i in range(self.n)]
+        visited[0] = True
+        edges = []
+
+        heap = []
+
+        curr = self.adj[0].head
+        while(curr is not None):
+            heapq.heappush(heap, (curr.weight, (0, curr.node)))
+            curr = curr.next
+        
+        while(nodes_left > 0):
+            (_, (u, v)) = heapq.heappop(heap)
+            edges.append((u, v))
+            visited[v] = True
+            nodes_left = nodes_left - 1
+
+            curr = self.adj[v].head
+            while(curr is not None):

```

Comment:
```suggestion
            while(curr):

```


A minor change according to PEP8 :)

Reply:
commented above about this